calblueprint/berkeley-student-cooperative
# Sami/shift card

# ✨ New in this PR ✨

## One liner: what did you do? 🛠
- created shiftCard component
- fixed shift query to include verification boolean
- created icon file in assets folder

## Coverage 🙆‍♀️
_use this section to break up your task into submodules and track progress_ 

## How can the reviewer test your code? 👩‍💻
- I put the shiftCard component into index so on the localHost you can create a shift -> will create a shift in the firebase
- leaving fields blank shouldn't let you create the shift? (we can create error fields in the future)
- number fields should only accept numbers

## Any bugs you encountered or still having trouble with? 🐛
let me know if there's any way I should clean up code, there's a lot of handler functions I made so not sure if that's the right practice

## Resources 📔
_link online resources and related PRs_ 


🏚 cc: @gregoriiaaa
